### PR TITLE
Update changelog and version

### PR DESCRIPTION
**Pull request by andreamari**
Step 2 of releasing guide: https://mitiq.readthedocs.io/en/latest/release.html#id3

### CODE REVIEW

File: CHANGELOG.md
Comment (crazy4pi314) [inline CHANGELOG.md:20]: I can't seem to find the `mitiq.pec.representations` submodule in the codebase, maybe I am just missing something, or maybe it is named something else?

**Review by crazy4pi314 — APPROVED** [review]
LGTM, I just had one comment on a package name. Probably should double check that, but I'll approve anyway to unblock. Thanks for being a great release manager @andreamari 💖

File: CHANGELOG.md
Comment (nathanshammah) [reply CHANGELOG.md:20]: @andreamari has this been renamed to https://github.com/unitaryfund/mitiq/blob/master/mitiq/pec/sampling.py?

**Review by nathanshammah — APPROVED** [review]

File: CHANGELOG.md
Comment (andreamari) [reply CHANGELOG.md:20]: I ment this folder: `https://github.com/unitaryfund/mitiq/tree/master/mitiq/pec/representations`
It also has an `__init__.py` file so I think it can be considered as a sub-module, right?

**Comment by andreamari** [comment]
I ment this folder: [https://github.com/unitaryfund/mitiq/tree/master/mitiq/pec/representations](https://github.com/unitaryfund/mitiq/tree/master/mitiq/pec/representations)
It also has an `__init__.py` file so I think it can be considered as a sub-module, right?

**Pull request merged**
